packages/menu/src/MenuItem.tsx

```diff
@@ -42,6 +42,14 @@ const titleTextStyle = css`
   font-weight: normal;
```

Comment:
Since we're already modifying this file, I don't think any of these rules do anything aside from `line-height` (which, despite it only shifting the text up 1-2px, it actually looks better without).

Reply:
👍